wp-content/themes/core/assets/pcss/media-queries/_variables.pcss

```diff
@@ -7,22 +7,26 @@
  *
  * ------------------------------------------------------------------------- */
 
-@custom-media --viewport-xxsmall (min-width: 320px);
-@custom-media --viewport-xsmall (min-width: 500px);
-@custom-media --viewport-small (min-width: 600px);
-@custom-media --viewport-small-max (max-width: 767px);
-@custom-media --viewport-medium (min-width: 768px);
-@custom-media --viewport-wpadmin (min-width: 783px);
```

Comment:
We should keep the wpadmin breakpoint defined (even if we're using it for tablet). Often designs don't stick to that value when they get implemented.

Reply:
I removed this because it no longer is true and now aligns with the `--mq-wp-medium` media query at the bottom of this file. Maybe the breakpoints have been updated in core, but `783px` is no longer mentioned in here: https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_breakpoints.scss.

I could rename `medium` to be `wpadmin` or `adminbar` so it's more in line with what it's used for generally?

Reply:
Interesting. Let's go with `adminbar`. We often use a different medium value on projects.

Reply:
Updated this variable to be `--mq-wp-adminbar`. This breakpoint will match the defined `tablet` media query but can be used in places where contexts dictates it makes more sense to reference the admin bar media query rather than the tablet media query.